app/com/github/pedrovgs/dotto/core/types/led.scala

```diff
@@ -1,10 +1,10 @@
-package com.github.pedrovgs.dotto.led
+package com.github.pedrovgs.dotto.core.types
 
 import scala.concurrent.duration.Duration
 
 sealed trait PinState
 
-case object High extends PinState
 case object Low extends PinState
+case object High extends PinState
 
-case class LedInteraction(state: PinState, duration: Duration)
+case class LedInteraction(state: PinState, duration: Duration)
```

Comment:
Good practice is to make case classes final as their inheritance is problematic due to it's syntetic generated methods.